### ISSUE TITLE
Add form to allow users to add remarks to their gateway

### ISSUE THREAD

**Issue by jdrmar** [TheThingsNetwork/gateway-status#1]
Currently I get emails with the required info, it would be much more convenient if people can update their gateways themselves (with some form of moderation preferably)

**Comment by gonzalocasas** [comment]
Isn't this partially implemented already? The packet forwarder takes description and contact email param in the config ([see here](https://github.com/TheThingsNetwork/packet_forwarder/blob/master/poly_pkt_fwd/global_conf.json#L250)). What I'm not sure if is this data is reaching the API or not yet.

**Comment by jdrmar** [comment]
Indeed, so packet forwarder -> api -> ttnstatus is the preferred way, not sure what the status on forwarder -> api is at the moment

**Comment by turiphro** [comment]
The /gateways view is a filtered set of fields, the /gateways/{EUI} shows
everything that's in Mongo. So if you're using the former, let me know
which Mongodb fields you would like to have added to the generic view.


Best,
Turiphro

**Comment by jdrmar** [comment]
Ah, ok, didn't know that. I'm seeing the following fields:

"datagramssent": 0,
"longitude": 6.5676,
"altitude": 6.0,
"datagramsreceived": 0,
"eui": "0000003197610001",
"ackratio": 100.0,
"rxcount": 0,
"latitude": 53.1338,
"time": "2015-11-30T10:10:46.041Z",
"rxforwarded": 0,
"rxok": 0

I'm guessing only latitude, longitude, (altitude) and time are relevant for the status, which are forwarded already.

**Comment by turiphro** [comment]
Weren't we talking about the extra fields Gonzalo mentioned? They're
probably not used that much yet since it's only in the latest commits of
TTN's repo.


Best,
Turiphro

**Comment by gonzalocasas** [comment]
Since this morning, we're using them on gateway FFFEB827EBA00A83, but I still don't see them in the API. Where are they getting lost?

**Comment by turiphro** [comment]
Probably in Croft; they're not in MongoDB. Does mentioning @johanstokking
work (from email)? ;-)


Best,
Turiphro

**Comment by gonzalocasas** [comment]
Now that you mention it, the issue is most likely in Jolie, 'cause Croft is pretty much just forwarding to the rabbitmq iirc.

**Comment by turiphro** [comment]
I keep having problems keeping apart Johan's two children ;)

**Comment by johanstokking** [comment]
@turiphro gotcha. Indeed, the current implementation doesn't store these fields. Please add an additional issue for that. I'd suggest that we come up with an API for this, apart from Croft and Jolie, as we're currently implementing the back-end core so we would need to start over again

**Comment by turiphro** [comment]
@johanstokking So where is all this core defining happening? As in: where
should the issue be added? I only see an empty 'docs' repo and the existing
jolie and croft.

**Comment by johanstokking** [comment]
@turiphro In docs, currently not transferred to our account yet. See @KtorZ
